contracts/registrar/ZNSRootRegistrarBase.sol

```diff
@@ -182,7 +166,7 @@ contract ZNSRootRegistrar is
             registry.createDomainRecord(args.domainHash, args.registrant, "address");
 
             IZNSAddressResolver(registry.getDomainResolver(args.domainHash))
-                .setAddress(args.domainHash, args.domainAddress);
+            .setAddress(args.domainHash, args.domainAddress);
```

Comment:
For formatting like this where lines from a single function call have to broken into multiple for the sake of line length, it makes more sense to have the extra indentation here. It shows visually that this line is not independent, I would undo this change and any other similar changes